sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePlugin.scala

```diff
@@ -51,6 +51,9 @@ object ScalaNativePlugin extends AutoPlugin {
     val nativeDump =
       settingKey[Boolean](
         "Shall native toolchain dump intermediate NIR to disk during linking?")
+
+    val optimise =
```

Comment:
Have you considered the American spelling `optimize`?

Reply:
I agree. Usually we use the American spelling of things in CS.

Reply:
Also, now is perhaps not a good time to add yet another settingKey, when we're in the process of merging them all in one `nativeConfig`.

Perhaps for now we can just do the `System.getenv` directly when building the `build.Config`.

Reply:
Given the choice, I usually prefer the British spelling.

(We also have other references to "optimiser" in the docs.)

cc @sjrd

Reply:
Maybe `Discover.optimize()` to central these env vars to be used by other build tools as well. Hate for everyone to duplicate logic

Reply:
@ekrich I like that as a solution.